ui-library-click-to-call/README.md

```diff
@@ -47,3 +53,1211 @@ Instead, it will copy all the configuration files and the transitive dependencie
+        horizontal
+        tokens={{ childrenGap: "1.5rem" }}
+        style={{ overflow: "hidden", margin: "auto" }}
+      >
+        <ClickToCallComponent
+          onRenderLogo={() => {
+            return (
+              <img
+                style={{ height: "4rem", width: "4rem", margin: "auto" }}
+                src={hero}
+                alt="logo"
+              />
+            );
+          }}
+          onSetDisplayName={setUserDisplayName}
+          onSetUseVideo={setUseVideo}
+        />
+      </Stack>
+    </Stack>
+  );
+};
+```
+This page has some general information on it for what our calling experiences can currently do. Also as we can see it is adding the widget component that we created earlier.
+
+Once we have made this we will need to add the new view to the root of the app `App.tsx`:
+
+`App.tsx`
+```typescript
+// add this with the other imports
+...
+import { ClickToCallScreen } from './views/ClickToCallScreen';
+...
+```
+
+```typescript
+    ...
+    switch (page) {
+        case 'click-to-call': {
+           if (!token || !userId || !locator) {
+            return (
+            <Stack verticalAlign='center' style={{height: '100%', width: '100%'}}>
+                <Spinner label={'Getting user credentials from server'} ariaLive="assertive" labelPosition="top" />;
+            </Stack>
+            )
+        }
+        return <ClickToCallScreen token={token} userId={userId} callLocator={locator} alternateCallerId={alternateCallerId}/>;
+    }
+    ...
+```
+
+Then if you have done this and set the arguments we defined in `App.tsx` you should see this when the app is running with `npm run start`:
+
+<img src='./tutorialImages/Sample-app-splash.png' width='800'>
+
+Then when you action the widget button you should see:
+
+<img src='./tutorialImages/Sample-app-widget-open.png' width='800'>
+
+Yay! We have made the control surface for the widget! Next we will talk about what we need to add to make this widget start a call in a new window.
+
+## Part 2 Creating a New Window Experience
+
+Now that we have a running application with our widget on the home page we will talk about starting the calling experience for your users with a new window. This scenario allows you to give your customer the ability to browse while still seeing your call in a new window. This can be useful in situations similar to when your users will use video and screensharing.
+
+To start we will create a new view in the `src/views` folder called `NewWindowCallScreen.tsx`. This new screen will be used by the `App.tsx` file to go into a new call with the arguments provided to it using our `CallComposite`. The `CallComposite` can be swapped with a stateful client and UI component experience if desired as well, but that will not be covered in this tutorial. See our [storybook documentation](https://azure.github.io/communication-ui-library/?path=/docs/quickstarts-statefulcallclient--page) for more information about the stateful client.
```

Comment:
```suggestion
To start we will create a new view in the `src/views` folder called `NewWindowCallScreen.tsx`. This new screen will be used by the `App.tsx` file to go into a new call with the arguments provided to it using our `CallComposite`. The `CallComposite` can be swapped with a stateful client and UI component experience if desired as well, but that will not be covered in this tutorial.
```

Remove storybook link due to beta vs stable version mismatch. We can discuss this further as a team on how to tackle this specific scenario "linking stateful client" storybook page to current features.

Reply:
I think that this should be fine, since its a stable dependency. I don't see a reason that this would break in the widget experience since they already did the setup when they started this tutorial with the beta flavor.